Movie_MVVM/MovieMVVMTests/KeychainTest.swift

```diff
@@ -0,0 +1,42 @@
+// KeychainTest.swift
+// Copyright © Avagyan Ernest. All rights reserved.
+
+@testable import Movie_MVVM
+import XCTest
+
+/// Тест кейчейн сервиса
+final class KeychainServiceTest: XCTestCase {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let emptyString = ""
+        static let testText = "Verification"
+        static let testApiString = "testApi"
+    }
+
+    // MARK: - Private Properties
+
+    private var keychainService: KeychainServiceProtocol?
+
+    // MARK: - Public Methods
+
+    override func setUpWithError() throws {
```

Comment:
```suggestion
    override func setUpWithError() throws {
```
если  override super ннада?